### PR TITLE
Create code of conduct & governance

### PR DESCRIPTION
**Pull request by isaacvetter**
Hey @NiklasSvenzen , @daliboz , @lbergnehr ,

Will you please review this proposed code of conduct and governance additions?

I think that this change should require unanimous committer approval. 

Isaac

p.s Niklas, are there really no hyphens/apostrophes in your name? (There aren't on your linkedin nor github profiles).

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+For small changes, at least one committer must approve the change. If a committer was the author of the change being reviewed, the approval must come from a different committer. For large, significant, or breaking changes, the committers must reach consensus on the change. Like *trivial*, the notion of *small* and *large* are subjective terms. However, a breaking change should be clear and unambiguous. Note that the size of the contribution does not determine whether the change requires just one approval or consensus. Rather, the impact of the change reflects the type of review required. For instance, the addition of a single optional field to the API would be considered a significant change and would warrant consensus amongst the committers.
+
+API changes are scrutinized far more closely than that of documentation or related code projects (like sample projects, test harnesses, etc). As such, API changes are always considered significant and require consensus from the committers.
```
Comment (lbergnehr) [inline GOVERNANCE.md]: We haven't talked much of the FHIRcast API. Should this be *specification* instead?

Comment (isaacvetter) [reply GOVERNANCE.md]: solid point, @lbergnehr ! Changed. 

Also, awesome review! 👍

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+## Voting and Consensus
+
+Anyone is welcome to comment on proposed changes. This includes expressing their opinion on why a proposed change should or should not be approved. Ultimately, committers make the final determination on accepting a proposed change.
+
+Committers should seek out the opinions and experiences from the broader community. It is the committers responsibility to balance the feedback from the community along with their own opinions and experiences.
+
+Committers must express their votes as:
+
+### :+1: Approve
```
Comment (daliboz) [inline GOVERNANCE.md]: Out of curiosity, if we're using the "review changes" functionality of the github PR, do we also need to do the +1/-1? Or is approving/request changes good for the +1/-1?

I can see that abstaining, if it needs to be a more solid "op out" than not doing a review, may require a more formal comment.

thoughts?

Comment (isaacvetter) [reply GOVERNANCE.md]: Hey @daliboz - nice, careful read!

I updated to better clarify that the formal github actions on a PR are equivalent to 👍 or 👎

**Pull request merged**